tests/NLog.UnitTests/Internal/StringBuilderExtTests.cs

```diff
@@ -74,5 +75,58 @@ void TestAppendInvariant(int input)
             StringBuilderExt.AppendInvariant(sb, input);
             Assert.Equal(input.ToString(System.Globalization.CultureInfo.InvariantCulture), sb.ToString());
         }
+
+        [Theory]
+        [MemberData(nameof(TestAppendXsdDateTimeRoundTripCases))]
+        void TestAppendXsdDateTimeRoundTrip(DateTime input)
```

Comment:
Proposal, 3 tests with same memberdata. 

(1 act is really prefered)

Reply:
Fixed